File: CS/MyMvpvmApplication/Views/TaskDetailView.cs
using System.Windows.Forms;
using DevExpress.Mvvm.POCO;
using MyMvpvmApplication.ViewModels;

namespace MyMvpvmApplication.Views {
    public partial class TaskDetailView : UserControl {
        public TaskDetailView() {
            InitializeComponent();
            viewModelCore = ViewModelSource.Create<TaskDetailViewModel>();
            presenterCore = new TaskPresenter(taskBindingSource, ViewModel);
            simpleButton1.BindCommand(() => ViewModel.ShowNotificaion(), ViewModel);
        }

        TaskDetailViewModel viewModelCore;
        public TaskDetailViewModel ViewModel {
            get { return viewModelCore; }
        }

        TaskPresenter presenterCore;
        public TaskPresenter Presenter {
            get { return presenterCore; }
        }
    }

    public class TaskPresenter {
        BindingSource bs;
        public TaskPresenter(BindingSource bs, TaskDetailViewModel viewModel) {
            viewModelCore = viewModel;
            this.bs = bs;
            ViewModel.TaskChanged += ViewModel_TaskChanged;
        }

        void ViewModel_TaskChanged(object sender, System.EventArgs e) {
             if (ViewModel.Task != null)
                 bs.DataSource = ViewModel.Task;
        }    
        
        TaskDetailViewModel viewModelCore;
        public TaskDetailViewModel ViewModel {
            get { return viewModelCore; }
        }
    }
}

File: VB/Views/TaskDetailView.vb

Imports System.Windows.Forms
Imports DevExpress.Mvvm.POCO
Imports MyMvpvmApplication.ViewModels

Namespace MyMvpvmApplication.Views
    Partial Public Class TaskDetailView
        Inherits UserControl
        Public Sub New()
            InitializeComponent()
            viewModelCore = ViewModelSource.Create(Of TaskDetailViewModel)()
            presenterCore = New TaskPresenter(taskBindingSource, ViewModel)
            simpleButton1.BindCommand(Sub() ViewModel.ShowNotificaion(), ViewModel)
        End Sub

        Private viewModelCore As TaskDetailViewModel
        Public ReadOnly Property ViewModel() As TaskDetailViewModel
            Get
                Return viewModelCore
            End Get
        End Property

        Private presenterCore As TaskPresenter
        Public ReadOnly Property Presenter() As TaskPresenter
            Get
                Return presenterCore
            End Get
        End Property
    End Class

    Public Class TaskPresenter
        Private bs As BindingSource
        Public Sub New(bs As BindingSource, viewModel__1 As TaskDetailViewModel)
            viewModelCore = viewModel__1
            Me.bs = bs
            AddHandler ViewModel.TaskChanged, AddressOf ViewModel_TaskChanged
        End Sub

        Private Sub ViewModel_TaskChanged(sender As Object, e As System.EventArgs)
            If ViewModel.Task IsNot Nothing Then
                bs.DataSource = ViewModel.Task
            End If
        End Sub

        Private viewModelCore As TaskDetailViewModel
        Public ReadOnly Property ViewModel() As TaskDetailViewModel
            Get
                Return viewModelCore
            End Get
        End Property
    End Class
End Namespace